File: String/Binary-String-Operation/binary_string_operation.py
def evaluate_binary_string(expr):
    result = int(expr[0])
    n = len(expr)

    for i in range(1, n - 1, 2):
        op = expr[i]
        val = int(expr[i + 1])

        if op == 'A':
            result &= val
        elif op == 'B':
            result |= val
        else:  # Treat 'C' as XOR
            result ^= val

    print("Result:", result)

# Example usage
expr = "1C0C1C1A0B1"
evaluate_binary_string(expr)

File: String/Binary-String-Operation/BinaryStringOperation.js
function evaluateBinaryString(expr) {
  let result = parseInt(expr[0])

  for (let i = 1; i < expr.length - 1; i += 2) {
    let op = expr[i]
    let val = parseInt(expr[i + 1])

    if (op === 'A') {
      result = result & val
    } else if (op === 'B') {
      result = result | val
    } else {
      // Treat 'C' as XOR
      result = result ^ val
    }
  }

  console.log('Result:', result)
}

// Example usage
evaluateBinaryString('1C0C1C1A0B1')
